rest/taskana-rest-spring/src/main/java/pro/taskana/RestConfiguration.java

```diff
@@ -15,19 +16,21 @@
 
 import pro.taskana.classification.api.ClassificationService;
 import pro.taskana.common.api.TaskanaEngine;
-import pro.taskana.common.rest.ldap.LdapClient;
 import pro.taskana.monitor.api.MonitorService;
 import pro.taskana.task.api.TaskService;
 import pro.taskana.workbasket.api.WorkbasketService;
 
 /** Configuration for REST service. */
 @Configuration
-@ComponentScan
+@ComponentScan("pro.taskana")
```

Comment:
Only the @ComponentScan annotation without arguments should be fine since this class already resides in the pro.taskana package

Reply:
I moved this class to `pro.taskana.common.rest`. Because of this this has to stay